wire-library/wire-grpc-server-generator/src/main/java/com/squareup/wire/kotlin/grpcserver/FileDescriptorGenerator.kt

```diff
@@ -0,0 +1,110 @@
+package com.squareup.wire.kotlin.grpcserver
+
+import com.google.protobuf.DescriptorProtos
+import com.google.protobuf.Descriptors
+import com.squareup.kotlinpoet.*
```

Comment:
nit: no star imports (we should probably add .editorconfig, but maybe not in this PR)